.github/workflows/pr-welcome-msg.yml

```
@@ -23,4 +23,5 @@ jobs:
             If you want to re-run tests or request review, you can use following commands as a comment:
             - **leapp-ci build** to run copr build and e2e tests in **OAMG CI**
             - **review please** to notify leapp developers of review request
+            Please [open ticket](https://url.corp.redhat.com/oamg-ci-issue) in case you experience technical problem with the CI. (RH internal only)
```

Add newlines for proper indentation/separation of the text from the list:
```suggestion

            Please [open ticket](https://url.corp.redhat.com/oamg-ci-issue) in case you experience technical problem with the CI. (RH internal only)
            
```